panther_manager/include/panther_manager/behavior_tree_utils.hpp

```diff
@@ -90,4 +90,29 @@ inline std::string GetLoggerPrefix(const std::string & bt_node_name)
 }
 }  // namespace panther_manager
 
+namespace BT
+{
+/**
+ * @brief Converts a string to a vector of integers.
+ *
+ * @param str The string to convert.
+ * @return std::vector<int> The vector of integers.
+ */
+template <>
+inline std::vector<int> convertFromString<std::vector<int>>(StringView str)
+{
+  auto parts = BT::splitString(str, ';');
+  if (!parts.size()) {
+    throw BT::RuntimeError("invalid input)");
```

Comment:
- Why not to throw `invalid_argument`?
- The message has a typo and can be more descriptive (why the input is invalid)

Reply:
The function is used inside BT tree so I think BT handles BT::RuntimeError as another `convertFromString` functions.